### DISCUSSION TITLE
is this customizable?

### DISCUSSION THREAD

**Discussion by diiv0** [SiimKirjanen/seatreg#77] [General]
![image](https://user-images.githubusercontent.com/32500966/210273458-14739b9e-62ad-4885-be74-eb9a9a6cf28b.png)

**Comment by SiimKirjanen** [comment]
Hey. Not really. How would you like to customize it?

**Reply by omtr** [reply]
i would like to rename some of the elements because i want to use it for a land area instead of a venue. we have a non profit animal shelter and want to start a crowdfunding campaign. people should be able to buy their symbolic land claim to support our plan to buy a piece of land in real life. so instead of seats it would be parcels of land in 10 square meter sizes.
any idea how i could change the ui?

another question: we plan to buy a land with about 3000 parcels. any idea how i could easily create the scheme from an existing map?

thank you very much for your help!

**Reply by SiimKirjanen** [reply]
The feature to customize the names are already in the to-do list. But I cant really say when it will be implemented. The only option currently for you is to manually change the texts in code.

Not sure if I understood your second question. If you have an existing map from the area then you can use is as a background image of the registration and drag the registration boxes to correct places.

**Reply by omtr** [reply]
Hi SimmKirjanen, thank you very much for your reply. Can you push me to the right direction which files i should touch to change the order form (the one where first/last name and email address are required.
2nd question: never mind, i figured out that i can load a background image, so all i had to do is to find the right scaling that fits to the block size.
awesome stuff!

**Comment by SiimKirjanen** [comment]
If you want to modify text then https://github.com/SiimKirjanen/seatreg/blob/master/php/seatreg_strings.php is the file.
But changing the first/last name and email address required logic is not so easy. Those fields are mandatory and cant be removed without making lot of changes.

**Reply by omtr** [reply]
yes, i already read that in another thread but i could just put in some dummy content and change the filed type ;)

**Reply by SiimKirjanen** [reply]
If you are brave enough then you should take a look at https://github.com/SiimKirjanen/seatreg/blob/master/registration/js/registration.js file. It has the logic that generates the booking form
